packages/langfuse-wrapper/scripts/setup_langfuse.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# setup_langfuse.sh
+#
+# This script sets up and runs Langfuse locally for development and testing purposes.
+#
+# Key functionalities:
+# 1. Downloads the latest docker-compose.yaml from the Langfuse repository
+# 2. Starts Langfuse using Docker Compose with default initialization variables
+# 3. Waits for the service to be available
+# 4. Launches a browser to open the local Langfuse UI
+# 5. Prints login credentials from the environment file
+#
+# Usage:
+#   ./setup_langfuse.sh
+#
+# Requirements:
+#   - Docker and Docker Compose
+#   - curl
+#   - A .env.langfuse.local file in the env directory
+#
+# Note: This script is intended for local development use only.
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LANGFUSE_DOCKER_COMPOSE_URL="https://raw.githubusercontent.com/langfuse/langfuse/main/docker-compose.yml"
+LANGFUSE_DOCKER_COMPOSE_FILE="$SCRIPT_DIR/docker-compose.yaml"
+LANGFUSE_ENV_FILE="$SCRIPT_DIR/../env/.env.langfuse.local"
+
+download_docker_compose() {
+    curl -o "$LANGFUSE_DOCKER_COMPOSE_FILE" "$LANGFUSE_DOCKER_COMPOSE_URL"
+}
+
+start_docker_compose() {
+    docker compose --env-file "$LANGFUSE_ENV_FILE" -f "$LANGFUSE_DOCKER_COMPOSE_FILE" up --detach
+}
+
+wait_for_service() {
```

Comment:
we might want to add a max retries guard since this has the potential for looping indefinitely. probably an edge case but nice little ux in the event it does!

:nit: `nc` gives minimal output, `curl` is a little handier esp since we're using the http context 

```sh
    local retries=30
    local count=0
    until curl -s http://localhost:3000 > /dev/null; do
        ((count++))
```